inference-engine/tests/functional/inference_engine/transformations/compare_functions_test.cpp

```diff
@@ -208,4 +210,267 @@ TEST(TransformationTests, CompareFunctoinsTINegative) {
     auto res = compare_functions(f, f_ref);
     EXPECT_FALSE(res.first);
     EXPECT_EQ(res.second, "LSTMCell/4 != Relu/0");
-}
\ No newline at end of file
+}
+
+TEST(TransformationTests, ConstantNegativeDifferentElementType) {
+    const auto createConstantFunc = [](ngraph::element::Type t) {
+        using namespace ngraph::opset5;
+        auto constant = Constant::create(t, ngraph::Shape{1}, {1.1});
+
+        return std::make_shared<ngraph::Function>(
+            ngraph::NodeVector{constant}, ngraph::ParameterVector{});
+    };
+
+    const auto& f1 = createConstantFunc(ngraph::element::f64);
+    const auto& f2 = createConstantFunc(ngraph::element::f32);
+
+    const auto& res = compare_functions(f1, f2, false, false, false, true, true);
+    EXPECT_FALSE(res.first);
+    EXPECT_THAT(res.second, HasSubstr("mismatch in value: 'element_type' : [f64] vs [f32]"));
+}
+
+TEST(TransformationTests, ConstantNegativeDifferentValues) {
+    const auto createConstantFunc = [](double value) {
+        using namespace ngraph::opset5;
+        auto constant = Constant::create(ngraph::element::f32, ngraph::Shape{1}, {value});
```

Comment:
nit: you use `using namespace ngraph::opset5;` but in any case you use it only in one place.

Reply:
I preferer to have `using namespace` declaration as narrow as possible. When it will be introduce globally it might affect other tests (some compilation ambiguous). Please convince me this is bas practice not only dummy repetition.

Reply:
I am ok to have `using namespace` declaration as narrow as possible. Moreover from my point of view it is a good practice. But I added this nitpick because for the current function we can avoid `using namespace` because we use `ngraph::opser5` only in order to create one `Constant` operation.